src/format.unit.ts

```diff
@@ -88,4 +88,40 @@ describe("format", () => {
 		let form = await model.types.Form.create({}) as any;
 		expect(form.toString("[Table]")).toBe("Text2, Text2");
 	});
+
+	test("Prefixes are not prepended if there is no value and result", async () => {
+		var model = new Model({
+			"Form": {
+				Name: {
+					label: "Name",
+					format: "[Prefix] [First] [MiddleInitial] [Last] [Suffix]",
```

Comment:
What happens if the format looks like this?
```js
"'[First]' '[Last]'"
```

Reply:
`'John'`
`Doe'`
`'`
Seems like an issue